------------------------------------------------------------
revno: 334
committer: Scott Moser <smoser@ubuntu.com>
branch nick: trunk
timestamp: Thu 2011-01-27 10:55:29 -0500
message:
  improve comment strings in rsyslog config file tools/21-cloudinit.conf
=== modified file 'ChangeLog'
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,5 +1,6 @@
 0.6.1:
  - fix bug in fixing permission on /var/log/cloud-init.log (LP: #704509)
+ - improve comment strings in rsyslog file tools/21-cloudinit.conf
 0.6.0:
  - change permissions of /var/log/cloud-init.log to accomodate
    syslog writing to it (LP: #704509)
--- a/tools/21-cloudinit.conf
+++ b/tools/21-cloudinit.conf
@@ -1,7 +1,6 @@
 # Log cloudinit generated log messages to file
 :syslogtag, isequal, "[CLOUDINIT]" /var/log/cloud-init.log
 
-# Uncomment the following to stop logging anything that matches the last rule.
-# Doing this will stop logging kernel generated UFW log messages to the file
-# normally containing kern.* messages (eg, /var/log/kern.log)
+# comment out the following line to allow CLOUDINIT messages through.
+# Doing so means you'll also get CLOUDINIT messages in /var/log/syslog
 & ~
